mkosi.md

```diff
@@ -306,8 +306,8 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
 
 : The distribution to install in the image. Takes one of the following
   arguments: `fedora`, `debian`, `ubuntu`, `arch`, `opensuse`,
-  `mageia`, `centos`, `clear`, `photon`, `openmandriva`. If not
-  specified, defaults to the distribution of the host.
+  `mageia`, `centos`, `clear`, `photon`, `openmandriva`, `rocky`,
```

Comment:
While you're at it, can you add `centos_epel` here? Just noticed that it is missing.

Reply:
Sure, no problem